lib/index.js

```diff
@@ -1,8 +1,10 @@
 import 'babel-polyfill';
 import Promise from 'bluebird';
+
+const __DEV__ = process && process.env && process.env.NODE_ENV === 'development';
```

Comment:
I took the pattern used in [lib/actions.js](https://github.com/elierotenberg/react-nexus/blob/v4-develop-2/lib/actions.js) and [lib/stores.js](https://github.com/elierotenberg/react-nexus/blob/v4-develop-2/lib/stores.js). I'm not sure there's something that injects `process.env.NODE_ENV` into the browser global object.